Docs/Configuration.tex

```diff
@@ -1469,42 +1452,42 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   This quirk requires \texttt{OC\_FIRMWARE\_RUNTIME} protocol implemented
   in \texttt{OpenRuntime.efi}.
 
-  \emph{Note}: This quirk can also be used as an ugly workaround to buggy UEFI
-  runtime services implementations that fail to write variables to NVRAM and
-  break the rest of the operating system.
+  \emph{Note}: This quirk can also be used as an ad-hoc workaround for flawed UEFI
+  runtime services implementations that are unable to write variables to NVRAM
+  and result in operating system failures.
 
 \item
   \texttt{DiscardHibernateMap}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Reuse original hibernate memory map.
 
-  This option forces XNU kernel to ignore newly supplied memory map and assume
-  that it did not change after waking from hibernation. This behaviour is required
-  to work by Windows, which mandates to
-  \href{https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-uefi#hibernation-state-s4-transition-requirements}{preserve}
+  This option forces the XNU kernel to ignore a newly supplied memory map and assume
+  that it did not change after waking from hibernation. This behaviour is required for
+   Windows to work. Windows mandates
+  \href{https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-uefi#hibernation-state-s4-transition-requirements}{preserving}
   runtime memory size and location after S4 wake.
 
-  \emph{Note}: This may be used to workaround buggy memory maps on older hardware,
-  and is now considered rare legacy. Examples of such hardware are Ivy Bridge laptops
-  with Insyde firmware, such as Acer V3-571G. Do not use this unless a complete understanding of
-  the consequences can be ensured.
+  \emph{Note}: This may be used to workaround problematic memory maps on older hardware
```

Comment:
"buggy" is less formal, but it's clearer what it means.

Reply:
Typically use "flawed" for "buggy". Will use that.

Reply:
cc @vit9696